views/settings/page-sections/dashboard.php

```diff
@@ -80,7 +80,14 @@
 						<span class="wpr-title3"><?php esc_html_e( 'License', 'rocket' ); ?></span>
 						<span class="wpr-infoAccount wpr-isValid" id="wpr-account-data">
 							<?php echo esc_html( $data['customer_data']['license_type'] ); ?>
-						</span><br>
+						</span>
+						<?php if ( ! $data['customer_data']['is_from_one_dot_com'] ) : ?>
+							<span>
+								<?php echo esc_html( 'with ' ); ?> 
```

Comment:
This needs to be translatable, you can use `esc_html_e()` function. I would also recommend to add the space directly in the HTML, not in the translation part, because it's not clear for translators.